autoapi/mappers/dotnet.py

```diff
@@ -98,8 +97,6 @@ def load(self, patterns, dirs, ignore=None, **kwargs):
                     self.app.warn(error_output)
             except (OSError, subprocess.CalledProcessError) as e:
                 self.app.warn('Error generating metadata: {0}'.format(e))
-                if raise_error:
-                    raise ExtensionError('Failure in docfx while generating AutoAPI output.')
```

Comment:
Similarly, I don't believe we can just remove this logic, any reason for it?

Reply:
I wasn't really sure what to do with this. I removed it because the linter was complaining that the addition of `**kwargs` meant that the function signature didn't match the base class. Ideally we want some way of setting `raise_error` through the base class rather than for dotnet specifically.
Perhaps we add a Werror/strict setting where warnings are raised as errors?

Reply:
What is the lint error on `**kwargs`? Seems we should keep this logic and work around the lint error instead.

Reply:
It complains that the method signature is different from the method on the base class, because we've added a new argument(s). I felt that adding a proper way of configuring whether to raise an error made more sense than adding kwargs. The extension interacts with the mappers in the same way for each language, so to me it makes more sense for the base class to be providing this type of configuration.

Reply:
Hrm, yeah not sure the best way to get around this. For now, feel free to revert, ignore the lint error, and open up an issue with some background so we can get back to addressing this more properly.